proto/rill/ai/v1/ai.proto

```diff
@@ -7,10 +7,16 @@ import "google/protobuf/struct.proto";
 message Tool {
   // The name of the tool
   string name = 1;
+  // The title of the tool
+  string title = 4;
```

Comment:
Nit: Maybe this should be `display_name` to follow our convention? Though I could see the argument that the `display_name` convention is only for our YAML API.

Reply:
Oh I see. I'm indifferent. Given I noticed it, I suppose I'd marginally lean `display_name`.